services/storage-users/pkg/config/defaults/defaultconfig.go

```diff
@@ -44,6 +44,7 @@ func DefaultConfig() *config.Config {
 		Reva:                    shared.DefaultRevaConfig(),
 		DataServerURL:           "http://localhost:9158/data",
 		DataGatewayURL:          "https://localhost:9200/data",
+		RevaGatewayGRPCAddr:     "127.0.0.1:9142",
```

Comment:
Are the `xxx_GATEWAY_GRPC_ADDR` not in format of `com.owncloud.api.gateway` ?
We had them formerly as IP address but changed them in summer to a name space...